plugins/src/messages.rs

```diff
@@ -63,6 +64,8 @@ pub struct GetManifestCall {}
 #[derive(Deserialize, Debug)]
 pub(crate) struct InitCall {
     pub(crate) options: HashMap<String, Value>,
+    pub(crate) configuration: Value,
+    // pub(crate) configuration: Configuration,
```

Comment:
If I try to put `Configuration` in here, then [this match](https://github.com/ElementsProject/lightning/blob/master/plugins/src/lib.rs#L188) isn't triggered. In principle it seems like this should work if everything is `Deserialize` ...